### PR TITLE
[layers/Radiotap] add HT, VHT field support

### PR DESCRIPTION
**Pull request by hkwi**
add support for 802.11n, 802.11ac fields

### CODE REVIEW

File: layers/radiotap.go [hunk 1]
```diff
@@ -42,6 +43,10 @@ const (
 	RadioTapPresentTxFlags
 	RadioTapPresentRtsRetries
 	RadioTapPresentDataRetries
+	_
+	RadioTapPresentMcs
```
Comment (gconnell) [inline layers/radiotap.go]: please capitalize acronyms:  MCS and VHT

File: layers/radiotap.go [hunk 2]
```diff
@@ -99,6 +104,15 @@ func (r RadioTapPresent) RtsRetries() bool {
 func (r RadioTapPresent) DataRetries() bool {
 	return r&RadioTapPresentDataRetries != 0
 }
+func (r RadioTapPresent) Mcs() bool {
```
Comment (gconnell) [inline layers/radiotap.go]: Ditto down here... MCS and VHT

File: layers/radiotap.go [hunk 3]
```diff
@@ -260,6 +274,335 @@ func (a RadioTapChannelFrequency) String() string {
...
+	if self.Known.Beamformed() {
+		if self.Flags.Beamformed() {
+			tokens = append(tokens, "beamformed")
+		} else {
+			tokens = append(tokens, "no beamformed")
+		}
+	}
+	if self.Known.Bandwidth() {
+		tokens = append(tokens, []string{
+			"20",
```
Comment (gconnell) [inline layers/radiotap.go]: Move this slice up into a global var:

var bandwidthStrings = []string{...}

File: layers/radiotap.go (hunk 1 above)
Comment (gconnell) [inline layers/radiotap.go]: In general, prefer capitalizing acronyms everywhere... Not sure if MCS and VHT are acronyms, but if so, capitalizing them and other new vars/functions/etc. with acronyms in them would be great.  We don't do it as consistently as we like, but try to do it for new code.

**Comment by gconnell** [comment]
Quick ping on this?

**Comment by hkwi** [comment]
I'd send patch within few days.

**Comment by hkwi** [comment]
I updated the patch.

**Pull request merged**